app/build.gradle

```diff
@@ -103,7 +104,8 @@ android {
                     //  - https://issuetracker.google.com/issues/36924841
                     //  - https://issuetracker.google.com/issues/36905922
                     "sharedUserId": "org.mozilla.firefox.sharedID",
-                    "deepLinkScheme": deepLinkSchemeValue
+                    "deepLinkScheme": deepLinkSchemeValue,
+                    "requestLegacyExternalStorage": true
```

Comment:
I see `"requestLegacyExternalStorage": true` being already set for `defaultConfig` from which we'll inherit it so maybe this is not needed. Also below.

Reply:
I just wanted to be extra careful as the whole array is being created here, avoid possible issues in case it gets deleted from the `defaultConfig`, and as a result leaking this value to stable versions.